README-usage.md

```diff
@@ -2,22 +2,34 @@
 
 ## ..MPyL CLI
 
-Install MPyL
+#### 1. Install MPyL
 
 ```shell
 pip install mpyl
 mpyl --help
 ```
 
-#### Health check
+#### 2. Health check
 
 ⭐ It is recommended to run this before running any other commands.
 ```shell
 mpyl health
 ```
 Will validate the configuration and check if all required tools are installed.
 
-##### Command structure
+#### 3. Create a PR with your changes
+
+```shell
+gh pr create --draft
+```
+
+#### 4. Trigger an MPyL build via the CLI
+
+```shell
+mpyl build jenkins
+```
```

Comment:
Maybe we could give options here. If you use GHA, the build will probably be triggered automatically. And you may also want to run a local build.